docs/application/vscode-ext/web.md

```diff
@@ -63,87 +63,160 @@ If you do not have a real device, you can run your application in a virtual envi
+      >
+      > - Linux OS: `/home/username/tizen-studio`
+
+      ![Change Tizen SDK Path](media/tizen_web24.png)
+
+### Install Tizen Baseline SDK
+
+> **Note**
+>
+> You must install Oracle Java Development Kit (JDK) 8 or OpenJDK 12 for Tizen Studio 3.5 and higher.
+>
+> For more information on download and install, see [Oracle Java Download page](https://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html).
+
+To install a Tizen Baseline SDK, follow these steps:
+
+1. In VS Code, click **View** > **Command Palette**.
+2. In the input field of the **Command Palette** that appears, type **Tizen Web: Install Tizen Baseline SDK** and press **Enter**.
+3. In the notification that appears, perform one of the following:
+    - If SDK is already installed, click **YES**, and run **Tizen Web: Change SDK Path**.
+
+      ![install check](media/tizen_web31.png)
+
+    - If SDK is not installed, to perform a fresh installation, click **NO**. In the notification that appears:
+
+      - To start the Tizen Baseline SDK installation, click **Install**.
+      - If the path of the popup is **null**, change the path by clicking **Change Path**.
+
+      ![install ask](media/tizen_web32.png)
+
+    Proceed with the installation. The installation progress appears in the log. The extension reloads when the installation is complete.
+
+    ![install run](media/tizen_web33.png)
+### Change Workspace Path
+
+To change a workspace path, follow thse steps:
+
+1. In VS Code, click **View** > **Command Palette**.
+2. In the input field of the **Command Palette** that appears, type **Tizen Web: Change Workspace Path** and press **Enter**.
+3. In the change notification that appears, perform one of the following:
+    - To keep the path, click **Keep**.
+    - To change the path, click **Change Path**. In the input field of the **Command Palette** that appears, type the path and press **Enter**.
+
+        > **Note**
+        >
+        > The default path for:
+        >
+        > - Window OS: `c:\workspace`
+        >
+        > - Linux OS: `/home/username/workspace`
+
+        ![Change Workspace Path](media/tizen_web25.png)
+
+### Change Chrome Path
+
+To change a Chrome browser path, follow these steps:
+
+1. In VS Code, click **View** > **Command Palette**.
+2. In the input field of the **Command Palette** that appears, type **Tizen Web: Change Chrome Path** and press **Enter**.
+3. In the change notification that appears:
+    - To keep the path, click **Keep**.
+    - To change the path, click **Change Path**. In the input field of the **Command Palette** that appears, type the path and press **Enter**.
+    
+    > **Note**
+    >
+    > If the installation is complete with the **Chrome** installer, the path is set to below default path:
+    >
+    > - Window OS: `C:\Program Files (x86)\Google\Chrome\Application\chrome.exe`
+    >
+    > - Linux OS: `/opt/google/chrome/google-chrome`
+
+    ![Change Chrome Path](media/tizen_web26.png)
```

Comment:
Please right align the note and the image with the bullet

Reply:
done.

Reply:
As this image is describing both line 140 and line 141, in my thought, current indentation is not bad.

(If the image is only for the line 141, it should be under 141. But the image is describing "Keep" and "Change Path" both.)